types/src/collection.rs

```diff
@@ -7,9 +7,28 @@ use {
 pub struct Collection {
     pub title: String,
 
-    pub wordpress_menu_id: i64,
+    pub slug: String,
+
+    pub wordpress_menu_id: Option<i64>,
 
     pub intro_chapters: Vec<Chapter>,
 
     pub genre_chapters: Vec<Chapter>
+}
+
+#[derive(Debug, Clone)]
+pub struct Chapter {
+    pub id: Option<Uuid>,
+    
+    pub url_slug: String,
+
+    pub index_in_parent: i64,
+
+    pub chapter_name: String,
+
+    pub document_short_name: Option<String>,
+
+    pub document_id: Option<DocumentId>,
```

Comment:
Why are these both fields here? I think you don't fill in `document_id` with `Some` in your code ever right?

Reply:
you're right-- document_id is not needed, got rid of it!